app/main.ts

```diff
@@ -166,6 +166,32 @@ function sendGameData(gameData: GameData): void {
   );
 }
 
+// Win condition
+
+function canPawnMove(pawn: Pawn): boolean {
+  if (!(gameData?.positionPawn == undefined)) {
+    let flag = false;
+    for (const pawnAdjacent of pawn.adjacent) {
+      const b = boxes[pawnAdjacent[1]][pawnAdjacent[0]];
+      if (!b.removed && !equalArray(pawnAdjacent, gameData.positionPawn[0]) && !equalArray(pawnAdjacent, gameData.positionPawn[1])) {
+        flag = true;
```

Comment:
you could remove the  `flag` variable and return here `true` directly and instead of writing `return flag;` you could write `return false`